db/revision_cache_lru.go

```diff
@@ -82,16 +82,17 @@ func (sc *ShardedLRURevisionCache) Remove(docID, revID string, collectionID uint
 
 // An LRU cache of document revision bodies, together with their channel access.
 type LRURevisionCache struct {
-	backingStores    map[uint32]RevisionCacheBackingStore
-	cache            map[IDAndRev]*list.Element
-	lruList          *list.List
-	cacheHits        *base.SgwIntStat
-	cacheMisses      *base.SgwIntStat
-	cacheNumItems    *base.SgwIntStat
-	lock             sync.Mutex
-	capacity         uint32
-	memoryCapacity   int64
-	cacheMemoryBytes *base.SgwIntStat
+	backingStores        map[uint32]RevisionCacheBackingStore
+	cache                map[IDAndRev]*list.Element
+	lruList              *list.List
+	cacheHits            *base.SgwIntStat
+	cacheMisses          *base.SgwIntStat
+	cacheNumItems        *base.SgwIntStat
+	lock                 sync.Mutex
+	capacity             uint32           // Max number of items capacity of LRURevisionCache
+	MaxMemoryCapacity    int64            // Max memory capacity of LRURevisionCache
```

Comment:
It doesn't look like this needs to be a public property (can be `maxMemoryCapacity`).  Actually - I'm not clear why this needed to change from `memoryCapacity`.

Reply:
Making public was mistake, the renaming was just to make it more clear in terms of distinction from `currMemoryCapacity` but after the rename of that to `currMemoryUsage` I can revert it.